kedro-org/kedro-viz
# Fix Demo dependencies

## Description

kedro and kedro-datasets on demo-project were pinned to 0.18.4 and 1.0.1 respectively. The preview feature is only available on kedro-datasets 1.2.0 hence I updated that in this PR

## Development notes



## QA notes

Create a clean conda environment. 
- conda create -n testdemo python=3.9

then do the below cmds in the demo_project folder
-  pip install -r src/demo_project/requirements.in 
- pip install -r src/docker_requirements.txt 
- pip install kedro-viz 
- kedro viz

The kedro-viz launched should have preview datasets showing. 

## Checklist
